tilepack/builder.py

```diff
@@ -158,6 +164,10 @@ def main():
     parser.add_argument('max_zoom',
         type=int,
         help='The maximum zoom level to include')
+    parser.add_argument('typ',
```

Comment:
Please change `typ` to `type` here so that help text isn't needlessly confusing. If `args.type` doesn't work, you can pass `destination='typ'` into the `add_argument()` call.